### ISSUE TITLE
TypeError: Cannot read property 'programStart' of undefined

### ISSUE THREAD

**Issue by z5h** [elm-lang/elm-repl#92]
In the REPL:
```
> import Time
> import Signal
> (Signal.constant 1) |> Time.timestamp
TypeError: Cannot read property 'programStart' of undefined
```

**Comment by amedeedaboville** [comment]
yeah I got this too when importing a module that used both Time and Signal. I'm a complete beginner in elm though so I have no idea what it means and I'll just work around it :/

**Comment by z5h** [comment]
@amedeedaboville  This is not a problem with the code input into the REPL. This is a problem with the REPL itself.

The REPL, if I understand correctly, does not have full support for Signals. Still, I'd expect to be able to be able to view my construction's type (for example) in the REPL without necessarily using it.

**Comment by magthe** [comment]
@z5h I agree, though an acceptable bare minimum might be to be able to view the types of *other* stuff in the module:

```
module Test where

import Time

foo = "bar"

mysig = Time.every Time.second
```

Due to `mysig` I can't even see the type of `foo` in *elm-repl*.

**Comment by jorgebg** [comment]
It is happening to me as well

**Comment by co-dh** [comment]
+1

**Comment by simonewebdesign** [comment]
Same issue.

```
> import Time
> Time.fps 60
TypeError: Cannot read property 'programStart' of undefined
```
